tests/release/pipelines/rh_advisories.go

```diff
@@ -97,6 +97,9 @@ var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for rh-advisories pip
 			if err = managedFw.AsKubeDeveloper.TektonController.StorePipelineRun(pipelineRun.Name, pipelineRun); err != nil {
```

Comment:
This doesn't make any sense anymore. You changed the code to never return an err, yet check here if an error is returned. That is why I asked to still return the error in the function

Reply:
We know from the CI run (that we discussed on the channel) that getting the logs from PipelineRun might fail - due to that I suggested Jing to just print out the error.

And there are still 2 other places in that method where the error can be returned, so why wouldn't make sense to check for the error here?